examples/chart/teleport-cluster/values.yaml

```diff
@@ -369,6 +369,10 @@ log:
 # Extra Kubernetes configuration #
 ##################################
 
+# nodeSelector to apply for pod assignment
+# https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
+nodeSelector: {}
```

Comment:
Not sure if we want to add a note about preferring the use of `affinity` instead? I suppose Kubernetes does officially support both so maybe it's not necessary.